CriticalMass/WaitOperation.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  WaitOperation.swift
+//  CriticalMaps
+//
+//  Created by Илья Глущук on 13/10/2019.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import UIKit
```

Comment:
Replacing Import UIKit with Import foundation should fix some of the SPM errors

Reply:
fixed, check it please